src/Sulu/Bundle/AdminBundle/Resources/js/components/Dialog/types.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+export type Action = {
```

Comment:
If the component above is not used we probably also don't need this type, right?